docs/user/how-to/use-cli.rst

```diff
@@ -0,0 +1,39 @@
+Use the command line interface (CLI)
+====================================
+
+Blueapi comes pacakged with a simple click based CLI. You can start the server
+and then query it for plans and devices, as well as ask it to run a plan, through
+the CLI.
+
+Starting the server
```

Comment:
This needs merging with the out of date cli docs in main (https://diamondlightsource.github.io/blueapi/main/user/how-to/run-cli.html) an requires a link to https://diamondlightsource.github.io/blueapi/main/user/reference/cli.html